src/irmin/irmin.mli

```diff
@@ -495,3 +495,11 @@ module Of_backend (B : Backend.S) :
 module Export_for_backends = Export_for_backends
 (** Helper module containing useful top-level types for defining Irmin backends.
     This module is relatively unstable. *)
+
+module Simple_storage = Simple_storage
+
+(** Key-value store creator using {!Simple_storage}. *)
+module Of_simple_storage
+    (M : Simple_storage.Maker)
+    (H : Hash.S)
+    (V : Contents.S) : KV with type hash = H.t and module Schema.Contents = V
```

Comment:
I think we should add somewhere (either here or in the exemple) that `Of_simple_storage` is an alternative to `Maker`, `KV_maker` and `Of_backend`.

Reply:
I updated the main documentation for this backend section to list the different ways to use the APIs.